samples/drivers/touch/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config CROSS_WIDTH
+	int "Horizontal length of the cross to be drawn"
+	default 20
+
+config CROSS_HEIGHT
+	int "Vertical length of the cross to be drawn"
+	default 20
```

Comment:
A nit but I think it would maybe make more sense to express size in terms of a ratio, this way there would be less chances people ever need to override the default value should their screen be really large/small.

Reply:
Switched to ratio